protopipe/pipeline/event_preparer.py

```diff
@@ -790,11 +799,7 @@ def prepare_event(self, source, return_stub=True, save_images=False, debug=False
             if self.LST_stereo and (n_triggered_LSTs < self.min_ntel_LST) and (n_triggered_LSTs != 0) and (not_LST_triggered_telescopes >= 2):
```

Comment:
The condition can be simpler I think. You have to set good_for_reco to 0 for all LSTs whenever  
self.LST_stereo and (n_triggered_LSTs < self.min_ntel_LST)

I'd say the other two conditions can be removed. Besides, the last one is misleading (do you mean "non-LST"?) Anyway, if there is only one telescope left, there won't be stereo reco, why do you need to check for it here?

Reply:
yes I mean non-LST

the >0 or !=0 is needed I'd say otherwise there is no point in looking for LSTs tel_ids no?

this check will never catch cases in which there is only 1 tel_id left (before taking into account the quality of each image which happens just after) because of the last condition

Reply:
the check wich catches 1 telescope left is the next one

`self.event_cutflow.cut("min2Tels reco", n_tels["GOOD images"])`